### PR TITLE
Update information-barriers.md

### PR DESCRIPTION
**Pull request by nibandyo**
Updated article with the changes for OneDrive info barriers per discussion with @kaarins

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: OneDrive/information-barriers.md [hunk 1]
```diff
@@ -37,9 +37,9 @@ When these segments are associated with the OneDrive, content can be shared with
...
-- [Complete the form](https://forms.microsoft.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR_mCQfArPllNo4NRhexFZxRUNEZCTTZLVVJHOEJOVVlCTUdJNDVERko5Qy4u) to enable SharePoint and OneDrive information barriers in your organization. 
 - Create segments and define the users in each. Create policies that block communication between the segments, and then set them to active. For info, see [Define policies for information barriers](/office365/securitycompliance/information-barriers-policies). Wait 24 hours for the changes to propagate through your environment.
-- After you have received an email response from us and set your information barrier policies to active, wait 24 hours for the changes to propagate through your environment.
+- [Complete the form](https://forms.microsoft.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR_mCQfArPllNo4NRhexFZxRUNEZCTTZLVVJHOEJOVVlCTUdJNDVERko5Qy4u) to enable SharePoint and OneDrive information barriers in your organization. Please note: SharePoint and OneDrive information barriers can only be enabled in your tenant if you have atleast one active information barriers policy defined in the tenant.
```
Comment (JohanFreelancer9) [inline OneDrive/information-barriers.md:41]:
```suggestion
- [Complete the form](https://forms.microsoft.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR_mCQfArPllNo4NRhexFZxRUNEZCTTZLVVJHOEJOVVlCTUdJNDVERko5Qy4u) to enable SharePoint and OneDrive information barriers in your organization. Please note that SharePoint and OneDrive information barriers can only be enabled in your tenant if you have at least one active information barriers policy defined in the tenant. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@kaarins PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**